polyjuice/polyjuice.py

```diff
@@ -124,17 +133,17 @@ def brew_potion(editor, working_file, out_dir, modifications, id_pairs, log):
                 ask_hermione(identified_folder)
                 dicom_folders.append(identified_folder)
 
-            editor.save_output(image, identified_folder, name)
+            editor.save_output(image, identified_folder, name, get_metadata)
             saving_message = "Saved to {}".format(identified_folder)
             log(saving_message)
 
-    except Exception, e:
+    except Exception as e:
         print("{} failed".format(name))
         failure_message = "{} failed".format(name) + "\n" + str(e)
         log(failure_message)
     return
 
-def add_hair(dicom_folders, zip_dir, log):
+def zip_files(dicom_folders, zip_dir, log):
```

Comment:
If you decide to keep this name for the function, the README needs to be edited.

Reply:
readme addressed